typescript/sdk/src/providers/SmartProvider/HyperlaneJsonRpcProvider.ts

```diff
@@ -32,6 +32,13 @@ export class HyperlaneJsonRpcProvider
     super(rpcConfig.connection ?? rpcConfig.http, network);
   }
 
+  prepareRequest(method: string, params: any): [string, any[]] {
+    if (method === ProviderMethod.MaxPriorityFeePerGas) {
+      return ['eth_maxPriorityFeePerGas', []];
+    }
```

Comment:
Why special case this?

Reply:
Just extending the functionality from the base json rpc provider in

https://github.com/ethers-io/ethers.js/blob/v5/packages/providers/src.ts/json-rpc-provider.ts#L555